apps/not-arrived/translations/en/default.json

```diff
@@ -1,1038 +1,358 @@
-{
```

Comment:
I think that should be in another PR that we raise after we've fixed all the translations. Otherwise, we'll be ignoring default.json files that have not be built by hof and that will break the build